lib/oci_utils/migrate/__init__.py

```diff
@@ -209,6 +271,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         """
         OciMigrateConfParam cleanup and exit.
         """
+        # placeholder
```

Comment:
if this is supposed to be overwritten by sub classes implementation and that's is 
mandatory , think about raising an exception or an assert

Reply:
no, it is useless here